packages/api/app/Models/User.js

```diff
@@ -38,17 +38,50 @@ class User extends Model {
 			role,
 			status,
 			company,
+			zipcode,
```

Comment:
Agora eu notei que toda esse código para criar a variável data não faz muito sentido. Podemos fazer somente `modelInstance.fill(payload);`

Aqui só precisa da lógica do `fullName` mesmo...

Reply:
Não dá certo fazer assim @nicholasio porque o payload possui o campo role, que não existe no modelInstance, na verdade o payload é tudo que chega independente de está no modelo ou não. Com relação ao data, eu vou construindo ela de acordo com o que for passado, se por exemplo eu não enviar um campo obrigatório, ex. status e eu simplesmente fizer `modelInstance.fill(data)` ele irá criar um campo null em status, sobrescrevendo o comportamento esperado que seria o status `pending`.